### PR TITLE
Style Iks Changes

### PR DESCRIPTION
**Pull request by IksGarG**

### CODE REVIEW

File: .github/workflows/build-and-test.yml
Comment (IksGarG) [inline .github/workflows/build-and-test.yml:1]: Need help @JoaquinBadillo

File: next.config.mjs
Comment (IksGarG) [inline next.config.mjs:1]: Also this! @JoaquinBadillo

File: package.json
Comment (IksGarG) [inline package.json:1]: Bye Bootstrap!

File: src/app/(main)/@agent/layout.tsx
Comment (IksGarG) [inline src/app/(main)/@agent/layout.tsx:1]: This for loading!

File: src/app/(main)/@agent/loading.tsx
Comment (IksGarG) [inline src/app/(main)/@agent/loading.tsx:1]: I did this for this: https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming

File: src/app/(main)/@agent/wait/page.tsx
Comment (IksGarG) [inline src/app/(main)/@agent/wait/page.tsx:1]: Deleted Bootstrap

File: src/app/(main)/@supervisor/dashboard/page.tsx
Comment (IksGarG) [inline src/app/(main)/@supervisor/dashboard/page.tsx:1]: Hidden RealtimeTranscript for styling purposes

File: src/app/(main)/@supervisor/layout.tsx
Comment (IksGarG) [inline src/app/(main)/@supervisor/layout.tsx:1]: Used Suspense for loading

File: next.config.mjs
Comment (JoaquinBadillo) [reply next.config.mjs:1]: Alright creo que esto lo integraste desde main entonces I like it

File: package.json
Comment (JoaquinBadillo) [reply package.json:1]: We are free

File: src/app/(main)/@agent/layout.tsx
Comment (JoaquinBadillo) [reply src/app/(main)/@agent/layout.tsx:1]: Yup yup suspense for the win

File: src/app/(main)/@agent/loading.tsx
Comment (JoaquinBadillo) [reply src/app/(main)/@agent/loading.tsx:1]: Thanks for the reference

File: src/app/(main)/@agent/wait/page.tsx
Comment (JoaquinBadillo) [reply src/app/(main)/@agent/wait/page.tsx:1]: <3

File: src/components/nav/navbar/index.tsx
Comment (JoaquinBadillo) [inline src/components/nav/navbar/index.tsx:1]: Muchos deletions, voy a testear esto en local

**Review by JoaquinBadillo — COMMENTED** [review]
Almost everything LGTM, nada más voy a correr un recorrido en local para garantizar funcionalidad

**Review by JoaquinBadillo — APPROVED** [review]
ALL GOOD!

**Pull request merged**